opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -550,15 +553,16 @@ export const stixLoadByIds = async (context, user, ids, opts = {}) => {
   }
   return ids.map((id) => loadedInstancesMap.get(id))
     .filter((i) => isNotEmptyField(i))
-    .map((e) => (convertStoreToStix(e)));
+    .map((e) => (convertStoreToStix_2_1(e)));
```

Comment:
Maybe it could be better for stixLoadByIds to also handle a version option.
convertStoreToStixWithResolvedFiles must also be handle to be accurrate depending of the stix version